src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -1764,12 +1766,12 @@ private IInterpolatedStringTextOperation CreateBoundInterpolatedStringTextOperat
 
         private IConstantPatternOperation CreateBoundConstantPatternOperation(BoundConstantPattern boundConstantPattern)
         {
-            Lazy<IOperation> value = new Lazy<IOperation>(() => Create(boundConstantPattern.Value));
-            SyntaxNode syntax = boundConstantPattern.Syntax;
-            ITypeSymbol type = null;
-            Optional<object> constantValue = default(Optional<object>);
-            bool isImplicit = boundConstantPattern.WasCompilerGenerated;
-            return new LazyConstantPattern(value, _semanticModel, syntax, type, constantValue, isImplicit);
+                Lazy<IOperation> value = new Lazy<IOperation>(() => Create(boundConstantPattern.Value));
+                SyntaxNode syntax = boundConstantPattern.Syntax;
+                ITypeSymbol type = null;
+                Optional<object> constantValue = default(Optional<object>);
+                bool isImplicit = boundConstantPattern.WasCompilerGenerated;
+                return new LazyConstantPattern(value, _semanticModel, syntax, type, constantValue, isImplicit);
```

Comment:
The offset change is unexpected

Reply:
This isn't a change here, but Resolved in [https://github.com/dotnet/roslyn/pull/23357/commits/758fd76932a6d9bcb640504c8fdef57fb5adecec](https://github.com/dotnet/roslyn/pull/23357/commits/758fd76932a6d9bcb640504c8fdef57fb5adecec) of [https://github.com/dotnet/roslyn/pull/23357](https://github.com/dotnet/roslyn/pull/23357)

---
In reply to: [156536612](https://github.com/dotnet/roslyn/pull/23209#discussion_r156536612) [](ancestors = 156536612)